Cesium.Core/Cesium.Core.csproj

```diff
@@ -0,0 +1,9 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
If we have Core, then `PreprocessorException` should be part of this exception infrastructure. We definitely would have some tricky preprocessor situations.

```
#define EOF 1
#define VALIDATE_EOF(code) if (code == EOF) { abort(-1); }
```

Most likely this will force us to store preprocessor AST as macro definitions and expand them. `VALIDATE_EOF` has `code` parameter which can be some other macro itself, and we probably should build tree and materialize it. I most likely would like to have WIP exception here.

Reply:
Addressed.